packages/legend-application-studio/src/components/editor/edit-panel/project-configuration-editor/ProjectConfigurationEditor.tsx

```diff
@@ -445,6 +433,237 @@ const ProjectDependencyEditor = observer(
   },
```

Comment:
Can you make sure we use `="..."` form for all of these?

Reply:
😓 🙇